inventory/sample/group_vars/k8s_cluster/k8s-net-calico.yml

```diff
@@ -109,6 +109,10 @@ calico_pool_blocksize: 26
 # calico_ip_auto_method: "interface=eth.*"
 # calico_ip6_auto_method: "interface=eth.*"
 
+# Set FELIX_MTUIFACEPATTERN, Pattern used to discover the host’s interface for MTU auto-detection.
+# see https://projectcalico.docs.tigera.io/reference/felix/configuration
+# calico_felix_mtu_iface_pattern: "^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*)"
```

Comment:
Lgtm according to https://projectcalico.docs.tigera.io/reference/felix/configuration